.backportrc.json

```diff
@@ -1,13 +1,14 @@
 {
   "targetBranchChoices": [
     { "name": "main", "checked": true },
+    "8.9",
     "8.8",
     "8.7"
```

Comment:
I think we could remove `8.7`, as there won't be any further patch release for `8.7`